receiptui/src/androidTest/java/com/hyperwallet/android/ui/receipt/ListPrepaidCardReceiptsTest.java

```diff
@@ -81,18 +82,22 @@ protected Intent getActivityIntent() {
                     return intent;
                 }
             };
+    private TimeZone mDefaultTimeZone;
 
     @Before
     public void setup() {
         mMockWebServer.mockResponse().withHttpResponseCode(HTTP_OK).withBody(sResourceManager
                 .getResourceContent("authentication_token_response.json")).mock();
 
+        mDefaultTimeZone = TimeZone.getDefault();
         setLocale(Locale.US);
+        TimeZone.setDefault(TimeZone.getTimeZone("PST"));
```

Comment:
I am okay with this kind of approach. But Here I think it may produce error for some time zone and value.
In this case activity will launch based on PST time zone. But still mock json response still GMT 0(there is no time zone defined. So default is GMT 0). I think We need to change mock response to PST time zone or here keep time zone as GMT 0

Reply:
@vinotb-altmtrk @skoong it only works when the time value below 8:30(24-hour format) from mock json. Because response is converting to USA/Pacific time from GMT. But the expected value still GMT. 
FYI: video attached in HW-60059.(I changed time for reproduce purpose)

Reply:
We are hardcoding the expected result now, so in your video you should be checking that its June 5th as GMT is 8 hours ahead of PST.